README.md

```diff
@@ -1,353 +1,130 @@
+
+
+[![demo of ArnePilot with this branch](https://img.youtube.com/vi/WKwSq8TPdpo/0.jpg)](https://www.youtube.com/playlist?list=PL3CGUyxys8DuTE1JTkdZwY93ejSfAGxyV)
+
+For a demo of this version of ArnePilot
+
+Find me on Discord https://discord.gg/Ebgn8Mr
+
+# Installation
+Put this URL in the custom URL field after uninstalling through the UI
+https://tinyurl.com/arne-dp
+or if you want to use the command line or https://github.com/jfrux/workbench
+`cd /data; rm -rf openpilot; git clone --depth 1 https://github.com/arne182/openpilot -b release5; reboot`
+
+#### Troubleshooting
+Arnepilot has comma logger disabled. This gives a 35% more cpu but at cost of giving [connection error](https://cdn.discordapp.com/attachments/538741329799413760/743231854764884067/image0.jpg)
+
+If you get the [no vehicle](https://cdn.discordapp.com/attachments/538741329799413760/743231854764884067/image0.jpg) after installing Arnepilot completely power cycle your device. If this still doesn't fix the problem look below at panda flashing and run the command. This is a known issue with comma2 users.
+
+## Panda flashing
+This is done automatically otherwise run `pkill -f boardd; cd /data/openpilot/panda/board; make; ` Make sure this done while your device is connnect to your car:
+- allowing no disengage on brake and gas for Toyota
+- changing acceleration limits for Toyota and
+- adapting lane departure warning where it gives you a slight push back into the middle of the lane without needing to be engaged (not yet complete)
+- The Panda version is also changed and checked.
+
+## Branches
+`release5`: this is the default branch that is most up to date with the ArnePilot 0.8 based off of [dragonpilot](https://github.com/dragonpilot-community/dragonpilot) release branch. This branch is early in development but will only get better
+
+`DP08-clean`: Current development branch.
+
+`release4`: this is my old branch, that is compatible with ArnePilot 0.7. Recommended branch until release5 gets better
+
+`release3`: this is my old branch, that is compatible with ArnePilot 0.6.
+
+`release2`: this is my old branch, that is compatible with ArnePilot 0.5.
+
+
+## Supported Cars
+Fork is known to work in both US and Europe
+- RAV4 Hybrid 2016-19
+- RAV4 2017-19
+- Corolla 2019-20
+- Prius 2017-2021
+- RX hyrid 2017
+- CT 2018
+- Chevrolet Volt 2017
+- Subaru Crosstrek Limited 2019 with 0.8 it will use Eyesight for radar. 
+
+### Todo
+- [ ] add Trafficd
+- [ ] improve tunning
+- [ ] add Dev UI
+- [ ] add OSM and Speed offset apk toggles
+- [ ] bring back feature op_edit from 0.7
+- [ ] Dynamic distance profiles
```

Comment:
df is already there

Reply:
we are stilling missing the event states for it